components/symbl_ai/actions/post-audio-url/post-audio-url.mjs

```diff
@@ -0,0 +1,126 @@
+      options: [
+        "default",
+        "phone",
+      ],
+      optional: true,
+    },
+    enableSeparateRecognitionPerChannel: {
+      type: "boolean",
+      label: "Enable Separate Recognition per Channel",
+      description: "Enables Speaker Separated Channel audio processing. Accepts `true` or `false` values.",
+      optional: true,
+    },
+    enableAllTrackers: {
+      type: "boolean",
+      label: "Enable All Trackers",
+      description: "Set this parameter to `true` to enable detection of all the Trackers configured for your account. Default value is `false`.",
+      optional: true,
+    },
+    enableSpeakerDiarization: {
+      type: "boolean",
+      label: "Enable Speaker Diarization",
+      description: "Set this parameter to `true` to enable Speaker Separation. Default value is `false`.  See [Speaker Separation](https://docs.symbl.ai/docs/async-api/overview/audio/post-audio-url/#speaker-separation) for reference.",
+      optional: true,
+    },
+    diarizationSpeakerCount: {
+      type: "string",
+      label: "Number of Speakers",
+      description: "The number of unique speakers in this conversation. See [Speaker Separation](https://docs.symbl.ai/docs/async-api/overview/audio/post-audio-url/#speaker-separation) for reference.",
+      optional: true,
+    },
+    trackers: {
+      type: "string",
+      label: "Trackers",
+      description: "Provide a JSON array of the information to be tracked containing the `name` and the `vocabulary` information. The tracker object is represented by the following structure: `[{\"name\": \"Promotion Mention\",\"vocabulary\": [\"We have a special promotion going on if you book this before\",\"I can offer you a discount of 10 or 20 percent you being a new customer for us\",\"We have a sale right now on\"]}]`. See doc [here](https://docs.symbl.ai/docs/management-api/trackers/create-tracker).",
+      optional: true,
+    },
+  },
+  async run({ $ }) {
+    const response =
+        await this.symblAIApp.postAudioUrl({
+          $,
+          data: {
+            url: this.audioUrl,
+            name: this.meetingName,
+            customVocabulary: this.customVocabulary,
+            confidenceThreshold: this.confidenceThreshold,
+            detectPhrases: this.detectPhrases,
+            webhookUrl: this.webhookUrl,
+            detectEntities: this.detectEntities,
+            languageCode: this.languageCode,
+            mode: this.mode,
+            enableSeparateRecognitionPerChannel: this.enableSeparateRecognitionPerChannel,
+            enableAllTrackers: this.enableAllTrackers,
+            enableSpeakerDiarization: this.enableSpeakerDiarization,
+            diarizationSpeakerCount: this.diarizationSpeakerCount,
+            trackers: JSON.parse(this.trackers),
+          },
+        });
```

Comment:
**Lint**

Decrease indentation by 2 spaces

Reply:
Fixed in 8c9deda4.